scripts/update_stubs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+SUBLIME_STUBS_DIR="$( cd "${SCRIPT_DIR}/../stubs/sublime_text" && pwd )"
+
+for filename in sublime sublime_plugin sublime_typing
+do
+    curl -L https://raw.githubusercontent.com/jfcherng-sublime/ST-API-stubs/master/${filename}.pyi --output "$SUBLIME_STUBS_DIR/${filename}.pyi"
```

Comment:
I have moved all those `*.pyi` file into a dedicated `typings/` folder. This should make scripting easier and more user friendly to find them all.

https://github.com/jfcherng-sublime/ST-API-stubs/tree/master/typings

Reply:
How are you suggesting to write the script now? I will just update the paths unless you have better suggestion

Reply:
After my test, yes... somehow my purpose doesn't work in a `TypedDict`.

Reply:
I think that we should put files under something like `resources/typings/sublime_text/*.pyi` because if I use pyright to edit this project (LSP-pyright), pyright will takes the `typings/` directory in this project but it's not used for type checking this project actually (at least, it's in a wrong directory structure).

Another possibility is that, if we want to bundle more stubs in the future, structure like `resources/typings/sublime_text/*.pyi` is more convenient since direct using `typings/` will make issue which I describe above.